roles/ceph-mon/tasks/docker/start_docker_monitor.yml

```diff
@@ -1,4 +1,7 @@
 ---
+- name: pull ceph daemon image
+  shell: "docker pull {{ ceph_mon_docker_username }}/{{ ceph_mon_docker_imagename }}"
```

Comment:
why do you pull the image? usually the run command below will take care of that.